packages/devtools_server/README.md

```diff
@@ -1,4 +1,8 @@
 ## What is this?
 
 This is a server that supports [Dart DevTools](https://pub.dartlang.org/packages/devtools).
-This package contains shared code for performing functions with DevTools, like launching it, for example.
\ No newline at end of file
+This package contains shared code for performing functions with DevTools, like launching it, for example.
```

Comment:
Perhaps, `This package contains shared code for performing functions with DevTools, for example, launching it.`

Reply:
Done.